src/Commands/InterfaceStartCommand.ts

```diff
@@ -0,0 +1,66 @@
+import { Command } from "./Command";
+import { CommandResult } from "./CommandResult";
+import { LineResults } from "./LineResults";
+import { Parameter } from "./Parameters/Parameter";
+import { RepeatingParameters } from "./Parameters/RepeatingParameters";
+import { SingleParameter } from "./Parameters/SingleParameter";
+
+/**
+ * A command for starting to declare an interface.
+ */
+export class InterfaceStartCommand extends Command {
+    /**
+     * Information on parameters this command takes in.
+     */
+    private static parameters: Parameter[] = [
+        new SingleParameter("InterfaceName", "The Interface name.", true),
+        new RepeatingParameters(
+            "Parent interfaces.",
+            [
+                new SingleParameter(
+                    "parentInterfaceName",
+                    "Name of parent interface.",
+                    true)
+            ])
+    ];
+
+    /**
+     * @returns Information on parameters this command takes in.
+     */
+    public getParameters(): Parameter[] {
+        return InterfaceStartCommand.parameters;
+    }
+
+    /**
+     * Renders the command for a language with the given parameters.
+     * 
+     * @param parameters   The command's name, followed by any parameters.
+     * @returns Line(s) of code in the language.
+     */
+    public render(parameters: string[]): LineResults {
+        let line: string = "";
+        let output: CommandResult[];
```

Comment:
I think you can declare `output` much below the following if statement, since it's not used until after.